ext/platform-inject-support/runtime/build.gradle.kts

```diff
@@ -15,13 +15,15 @@
  */
 
 repositories {
+  maven("https://jitpack.io/")
   maven("https://repo.waterdog.dev/releases/")
   maven("https://repo.md-5.net/repository/releases/")
   maven("https://repo.md-5.net/repository/snapshots/")
   maven("https://repo.opencollab.dev/maven-releases/")
   maven("https://repo.opencollab.dev/maven-snapshots/")
   maven("https://repo.papermc.io/repository/maven-public/")
   maven("https://hub.spigotmc.org/nexus/content/repositories/snapshots/")
+  maven("https://repo.loohpjames.com/repository")
```

Comment:
please sort the repos by length and for what do we need jitpack, we usually try to avoid them due to stability issues

Reply:
Will change jitpack to maven.scijava.org. Limbo depends on NBT lib from Querz 5.5, which is only on jitpack, while scijava has a 6.1 version.

Reply:
@derklaro can we maybe mirror the dependency